src/components/team/TeamPerson.tsx

```diff
@@ -0,0 +1,52 @@
+import Image from "next/image";
+import classNames from "classnames/bind";
+import styles from "./TeamPerson.module.css";
+import { ITeamPerson } from "./types";
+
+const cx = classNames.bind(styles);
+
+function Person({ name, role, photo }: ITeamPerson) {
+  return (
+    <figure className={cx("team-person__inner")}>
+      {photo.length > 0 && (
+        <div className={cx("team-person__photo")}>
+          <Image src={photo} width={300} height={300} alt="" />
```

Comment:
No, it's better not to use this thing :(
We often go beyond the limits of Vercel.